README.md

```diff
@@ -1,9 +1,77 @@
-# Coming soon
+# Run Salesforce Code Analyzer - GitHub Action
 
 [![GitHub Super-Linter](https://github.com/actions/typescript-action/actions/workflows/linter.yml/badge.svg)](https://github.com/super-linter/super-linter)
 ![CI](https://github.com/actions/typescript-action/actions/workflows/ci.yml/badge.svg)
 [![Check dist/](https://github.com/actions/typescript-action/actions/workflows/check-dist.yml/badge.svg)](https://github.com/actions/typescript-action/actions/workflows/check-dist.yml)
 [![CodeQL](https://github.com/actions/typescript-action/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/actions/typescript-action/actions/workflows/codeql-analysis.yml)
 [![Coverage](./badges/coverage.svg)](./badges/coverage.svg)
 
-This GitHub action will be available in the coming weeks. When available, we will update this README.md file.
+The `run-code-analyzer` GitHub Action scans your code for violations using 
+[Salesforce Code Analyzer](https://forcedotcom.github.io/sfdx-scanner), uploads the results as an artifact, and displays
+the results as a job summary.
+
+# Inputs
+* `run-command` (Default value: `'run'`)
```

Comment:
@stephen-carter-at-sf I just noticed something that we should update throughout this file: we don't need any single quotes around code - just code case like `run` and not `'run'` (code case and quotes). Do you want me to comment on each time I see this, or could you updated all to code case without also having single quotes? Thanks!